metatron/src/test/java/uk/gov/ida/eidas/metatron/apprule/rules/MetatronTests.java

```diff
@@ -82,7 +85,12 @@ public void firstValidResolvesOk() {
         Response response = getClient().target(getUriString(entityId)).request().get();
 
         assertThat(response.getStatusInfo()).isEqualTo(Response.Status.OK);
-        assertThat(response.readEntity(String.class)).isEqualTo(entityId);
+        CountryMetadataResponse actual = response.readEntity(CountryMetadataResponse.class);
```

Comment:
Could this be moved up to under the call to `getClient()`, so all the assert are grouped at the end?

Reply:
Yes, done